hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/TaskTimesService.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.recon.api;
+
+import org.hadoop.ozone.recon.schema.tables.daos.ReconTaskStatusDao;
+import org.hadoop.ozone.recon.schema.tables.pojos.ReconTaskStatus;
+import org.jooq.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+/**
+ * Endpoint for displaying the last successful run of each Recon Task
+ */
+@Path("/task")
+@Produces(MediaType.APPLICATION_JSON)
+public class TaskTimesService {
+  private static final Logger LOG =
+      LoggerFactory.getLogger(TaskTimesService.class);
+
+  private ReconTaskStatusDao reconTaskStatusDao;
+
+  @Inject
+  private Configuration sqlConfiguration;
+
+  ReconTaskStatusDao getDao() {
+    return reconTaskStatusDao == null ? new
+        ReconTaskStatusDao(sqlConfiguration) : reconTaskStatusDao;
+  }
+
+  /**
+   * Return the list of Recon Tasks and the last successful timestamp and
+   * sequence number.
+   * @return {@link Response}
+   */
+  @GET
+  @Path("taskStatus")
```

Comment:
Can we change this path to just "status"?

Reply:
Done.